infection.json.dist

```diff
@@ -10,6 +10,12 @@
             "branch": "master"
         }
     },
+    "mutators": {
+        "@default": true,
+        "global-ignoreSourceCodeByRegex": [
+            "assert\\(\\$issuedAt instanceof DateTimeImmutable\\);"
```

Comment:
@Ocramius in order to backport the bugfix for https://github.com/psr7-sessions/storageless/pull/146#discussion_r547784717 I basically had to  backport everything that doesn't break BC.
Once this gets merged, I'll rebase https://github.com/psr7-sessions/storageless/pull/146
This line here is the only one I've left sort of unaddressed from what you asked me, as I can't handle it right now; but to me seems just a minor improvement that can be postponed.

Reply:
That's acceptable, no worries :+1: